flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/TableConfigOptions.java

```diff
@@ -233,6 +233,19 @@ private TableConfigOptions() {}
                     .withDescription(
                             "Specifies a threshold where class members of generated code will be grouped into arrays by types.");
 
+    @Documentation.ExcludeFromDocumentation(
```

Comment:
I question whether this should be excluded from the documentation. I suggest this would be better to be documented with the proviso not to change this in most circumstances. The docs should also indicate the symptom the user sees if this option is needed to be increased -> so if this is hit then the solution can be readily found.

Reply:
My thought is that these parameters are rarely used, and in most cases, this type of issue does not occur. Therefore, we have this annotation `@Documentation.ExcludeFromDocumentation`.
Finally, just like the previous parameter `MAX_MEMBERS_GENERATED_CODE`, the same annotation is maintained here.